content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -0,0 +1,85 @@
+related:
+  - /influxdb/cloud/api/#tag/Write, InfluxDB API /write endpoint
+  - /influxdb/cloud/reference/internals
+  - /influxdb/cloud/reference/cli/influx/write
+---
+
+Learn how to handle and recover from errors when writing to InfluxDB.
+
+- [Discover common failure scenarios](#common-failure-scenarios)
+- [HTTP status codes](#http-status-codes)
+- [Troubleshoot failures](#troubleshoot-failures)
+
+## Common failure scenarios
+
+InfluxDB write requests may fail for a number of reasons.
+Common failure scenarios that return an HTTP `4xx` or `5xx` error status code include the following:
+
+- Exceeded a rate limit.
+- API token was invalid.
+- Client or server reached a timeout threshold.
+- Size of the data payload was too large.
+- Data was not formatted correctly.
+- Data did not conform to the [explicit bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/).
+  See how to troubleshoot specific [bucket schema errors](/influxdb/cloud/organizations/buckets/bucket-schema/#troubleshoot-errors).
+
+Writes may fail partially or completely even though InfluxDB returns an HTTP `2xx` status code for a valid request.
+For example, a partial write may occur when InfluxDB writes all points that conform to the bucket schema, but rejects points that have the wrong data type in a field.
+
+## HTTP status codes
+
+InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
+Write requests return the following status codes:
+
+- `204` **Success**: InfluxDB validated the request data format and accepted the data for writing to the bucket.
+    {{% note %}}
+  `204` doesn't indicate a successful write operation since writes are asynchronous.
+  If some of your data did not write to the bucket, see how to [troubleshoot rejected points](#troubleshoot-rejected-points).
+    {{% /note %}}
+
+- `400` **Bad request**: The line protocol data in the request was malformed.
+   The response body contains the first malformed line in the data. All request data was rejected and not written.
+- `401` **Unauthorized**: May indicate one of the following:
+  - `Authorization: Token` header is missing or malformed.
+  - API token value is missing from the header.
+  - API token does not have sufficient permissions to write to the organization and bucket.
+- `404` **Not found**: A requested resource (e.g. an organization or bucket) was not found. The response body contains the requested resource type, e.g. "organization", and resource name.
+- `413` **Request entity too large**: The payload exceeded the 50MB limit. All request data was rejected and not written.
+- `429` **Too many requests**: API token is temporarily over quota. The `Retry-After` header describes when to try the write request again.
+- `500` **Internal server error**: Default HTTP status for an error.
+- `503` **Service unavailable**: Server is temporarily unavailable to accept writes. The `Retry-After` header describes when to try the write again.
```

Comment:
For each response codes, maybe provide some instruction on what to do to resolve. For example:

- link to review the correct format for line protocol.
- link to how/where to retrieve the API token
- where to view resources
- how to split large payloads, by time or other filter?
- anything more helpful to trace internal server errors?